### DISCUSSION TITLE
Support PostgreSQL materialized view.

### DISCUSSION THREAD

**Discussion by zKoz210** [Soyhuce/next-ide-helper#82] [Ideas] [answered]
Lacks materialized view support for PostgreSQL. An example of a table in a database:

<img width="370" alt="image" src="https://user-images.githubusercontent.com/28255085/233779445-f1bad590-2382-494d-85ef-4f9b2b187a23.png">

Result:
<img width="305" alt="image" src="https://user-images.githubusercontent.com/28255085/233779458-5380dac4-a45c-40ba-84f1-e5b62c441642.png">

**Comment by bastien-phi** [comment]
Can you create a new laravel project with this issue ? I would be easier for me to fix this

**Reply by zKoz210** [reply]
I'm pretty busy creating a new project. if I give you an example of SQL table creation, can you test it?

```
create table public.purchases (
    id integer,
    name varchar(255),
    amount integer
);

insert into public.purchases (id, name, amount) VALUES (1, 'Test1', 250);
insert into public.purchases (id, name, amount) VALUES (2, 'Test2', 3000);
insert into public.purchases (id, name, amount) VALUES (2, 'Test3', 5000);

create materialized view shop_statistics as
    select sum(amount) from public.purchases as sum_all_purchases;

refresh materialized view shop_statistics;
```

f you create a ShopStatistics model, no documentation is generated for this column/columns

**Reply by zKoz210** [reply]
https://github.com/doctrine/dbal/issues/5821

**Comment by bastien-phi** [comment, marked as answer]
It looks like that was implemented in https://github.com/Soyhuce/next-ide-helper/pull/110
